samples/SmokeTest/app.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
```

Comment:
Any reason you have both the js and the ts files committed? I would have expected only the ts files.

Reply:
No particular reason, I'm going to remove the js file.